src/components/_LuxSubscribeNewsletter.vue

```diff
@@ -15,7 +15,7 @@
           type="email"
           name="EMAIL"
           aria-required="true"
-          aria-label="Subscribed"
+          aria-label="Enter your email address"
```

Comment:
Could we keep it more concise?
```suggestion
          aria-label="Email address"
```